libkineto/include/IActivityProfiler.h

```diff
@@ -81,6 +81,8 @@ class IActivityProfilerSession {
   // stop the trace collection synchronously
   virtual void stop() = 0;
 
+  virtual void reset() = 0;
```

Comment:
can you add a comment what the reset() does? Just a quick question, why have a reset() why not create a new session that starts with fresh state?

Reply:
I have removed this addition. It is really not needed.